src/makeInfinityAdder.js

```diff
@@ -1,10 +1,24 @@
 'use strict';
 
-/***
+/*
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  return function adder(number) {
+    if (number === undefined) {
```

Comment:
Consider adding a check to ensure that `number` is a valid numeric input. If `number` is not a number, you might want to throw an error or ignore it to prevent unexpected behavior.